README.md

```diff
@@ -110,174 +113,148 @@ server.start((err) => {
 
 ## API
 
-### Daemon Factory
+### Daemon Factory Class
 
-#### Create a `DaemonFactory` - `const df = DaemonFactory.create([options])`
+#### `DaemonFactory` - `const df = DaemonFactory.create([options])`
 
-> `DaemonFactory.create([options])` returns an object that will expose the `df.spawn` method
+`DaemonFactory.create([options])` returns an object that will expose the `df.spawn` method
 
 - `options` - an optional object with the following properties
   - `remote` bool - indicates if the factory should spawn local or remote nodes. By default, local nodes are spawned in Node.js and remote nodes are spawned in Browser environments.
   - `port` number - the port number to use for the remote factory. It should match the port on which `DaemonFactory.server` was started. Defaults to 9999.
-  - type - the daemon type to create with this factory. See the section bellow for the supported types
+  - `type` - the daemon type to create with this factory. See the section bellow for the supported types
+  - `exec` - path to the desired IPFS executable to spawn, otherwise `ipfsd-ctl` will try to locate the correct one based on the `type`. In the case of `proc` type, exec is required and expects an IPFS coderef.
```

Comment:
@dryajov seems that you never got to update the README based on the new reviewed API. I've updated it for you, please confirm that these options are correct.